src/main/java/com/labsynch/labseer/utils/PostSuccessfulAuthenticationHandler.java

```diff
@@ -0,0 +1,59 @@
+package com.labsynch.labseer.utils;
+
+import java.io.IOException;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.ldap.userdetails.LdapUserDetails;
+import org.springframework.security.web.authentication.SavedRequestAwareAuthenticationSuccessHandler;
+import org.springframework.security.web.savedrequest.HttpSessionRequestCache;
+import org.springframework.security.web.savedrequest.SavedRequest;
+
+public class PostSuccessfulAuthenticationHandler extends  SavedRequestAwareAuthenticationSuccessHandler {
```

Comment:
Has anything changed about how ACAS Roo and CReg do authentication? Is this file still necessary, and does it need any changes?

Reply:
Nothing about ACAS ROO or creg has changed with regards to authentication, this file works and should be kept in place.